client/src/pages/Home.tsx

```diff
@@ -341,10 +358,9 @@ function Home(this: any) {
               onClick={getSentiment}
               style={{
                 backgroundColor: "#2e7faa",
-                cursor: isButtonDisabled ? "not-allowed" : "pointer",
               }}
               className="mt-8 ml-52 py-2 px-4 text-white rounded"
-              disabled={isButtonDisabled}
+              // disabled={isButtonDisabled}
```

Comment:
The Cypress test is failing because this button is no longer disabled?

Reply:
@gracebilliris this was one of the criteria 
 The Sentiment button can be pressed when the Text to be Summarised is empty.

Reply:
We discussed this on discord but for the record: it wasn’t a criteria - it was bug. Something that shouldn’t work but does!